### PR TITLE
Update Changelog for v3.24.0

### PR DESCRIPTION
**Pull request by snewcomer**
- [x] Update Changelog
- [x] Ember packages bump to their latest release
- [x] Bump uglify to terser to fix fatal CI issues.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+- [#7359](https://github.com/emberjs/data/pull/7359) [CI]: bump failure asset size to 75 bytes
 - [#7369](https://github.com/emberjs/data/pull/7369) [CHORE] Bump @ember/ordered-set
 - [#7370](https://github.com/emberjs/data/pull/7370) Don't expose the value of a model's attribute in assertions (#7370)
-- [#7363](https://github.com/emberjs/data/pull/7363) [BUGFIX identifiers] Address issue with polymorphic findRecord  (#7363)
+- [#7363](https://github.com/emberjs/data/pull/7363) [BUGFIX identifiers] Address issue with polymorphic findRecord
+- [#7262](https://github.com/emberjs/data/pull/7262) [CHORE]: Extract internalModel access to identifiers for Relationships
+
+## Release 3.23.0 (November 30, 2020)
+
+## Release 3.22.0 (November 5, 2020)
```
Comment (snewcomer) [inline CHANGELOG.md:13]: I was having trouble figuring out what should go in these slots

Comment (igorT) [reply CHANGELOG.md:13]: The release channel, should have them we should backport

Comment (igorT) [reply CHANGELOG.md:13]: Good call https://github.com/emberjs/data/pull/7383/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR3

Comment (igorT) [reply CHANGELOG.md:13]: https://github.com/emberjs/data/pull/7383/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR10
You can copy paste

**Review by igorT — APPROVED** [review]

**Pull request merged**